apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -804,4 +996,438 @@ export const notImplementedCards: DbCard[] = [
 		attack: 12,
 		effects: {},
 	},
+	{
+		id: '51',
+		name: 'Magic Shield of Flame',
+		type: 'spell',
+		colors: ['red'],
+		attack: 5,
+		description: 'If this spell is beaten, negate the effect of the card that beat it. (NOT IMPLEMENTED)',
+		effects: {},
+	},
+	{
+		id: '52',
+		name: 'Uncast',
+		type: 'spell',
+		colors: ['blue'],
+		attack: 0,
+		description: 'Whener this spell is revealed, remove all cards being revealed this turn from play. (NOT IMPLEMENTED)',
+		effects: {},
+	},
+	{
+		id: '53',
+		name: 'Fields of mana',
+		type: 'field',
+		color: 'blue',
+		description: 'During cast phase, players can cast one more card. Players cannot cast two cards for the same slot.',
+		effects: {},
+	},
+	{
+		id: '54',
+		name: 'Blue wand',
+		type: 'spell',
+		colors: ['blue'],
+		attack: 8,
+		description: 'BLUE field effects trigger twice',
+		effects: {},
+	},
+	{
+		id: '56',
+		name: 'Barren Land',
+		type: 'field',
+		color: null,
+		description: 'Spells have no effects',
+		effects: {},
+	},
+	{
+		id: '64',
+		name: 'Bull’s eye shot',
+		type: 'spell',
+		colors: [],
+		attack: 0,
+		description: '4X Attack, where X is the number of  NEUTRAL cards in play.',
+		effects: {},
+	},
+	{
+		id: '65',
+		name: 'Tsunami',
+		type: 'spell',
+		colors: ['blue'],
+		attack: 0,
+		description: 'X Attack, where X is the combined number of every player’s BLUE spell stacks',
+		effects: {},
+	},
+	{
+		id: '66',
+		name: 'Community Hall',
+		type: 'field',
+		color: null,
+		description: 'The player with the fewest HP draws one more card during the draw phase',
+		effects: {},
+	},
+	{
+		id: '67',
+		name: 'Hilltop observatory',
+		type: 'field',
+		color: null,
+		description: 'The player with the lowest combined stack of spells keeps their hand revealed',
+		effects: {},
+	},
+	{
+		id: '68',
+		name: 'Back to Square One',
+		type: 'field',
+		color: null,
+		description: 'All spells have 10 Attack',
+		effects: {},
+	},
+	{
+		id: '69',
+		name: 'Upheaval',
+		type: 'spell',
+		colors: [],
+		attack: 25,
+		description: 'If Field Spell Stack == 0 Else 0 Attack',
+		effects: {},
+	},
+	{
+		id: '70',
+		name: 'Eldritch Blast',
+		type: 'spell',
+		colors: ['blue', 'green', 'red'],
+		attack: 10,
+		description: '',
+		effects: {},
+	},
+	{
+		id: '71',
+		name: 'Agonising Blast',
+		type: 'spell',
+		colors: ['red'],
+		attack: 10,
+		description: 'If Eldritch Blast (https://www.notion.so/Eldritch-Blast-3892a7ae6830411da392903c16ea5271?pvs=21) is in play on your side of the field, add +10 Attack to Eldritch Blast',
```

Comment:
notion page HAHAHA

Reply:
HAHAHAH listen, I just automated the CSV to TS code hahaha 

Will remove!